specification/purview/data-plane/Azure.Purview.Catalog/preview/2020-12-01-preview/purviewcatalog.json

```diff
@@ -0,0 +1,6748 @@
+            },
+            "replacementTerms": {
+              "type": "array",
+              "description": "An array of related term headers for replacement.",
+              "items": {
+                "$ref": "#/definitions/AtlasRelatedTermHeader"
+              }
+            },
+            "seeAlso": {
+              "type": "array",
+              "description": "An array of related term headers for see also.",
+              "items": {
+                "$ref": "#/definitions/AtlasRelatedTermHeader"
+              }
+            },
+            "synonyms": {
+              "type": "array",
+              "description": "An array of related term headers as synonyms.",
+              "items": {
+                "$ref": "#/definitions/AtlasRelatedTermHeader"
+              }
+            },
+            "translatedTerms": {
+              "type": "array",
+              "description": "An array of translated related term headers.",
+              "items": {
+                "$ref": "#/definitions/AtlasRelatedTermHeader"
+              }
+            },
+            "translationTerms": {
+              "type": "array",
+              "description": "An array of related term headers for translation.",
+              "items": {
+                "$ref": "#/definitions/AtlasRelatedTermHeader"
+              }
+            },
+            "usage": {
+              "type": "string",
+              "description": "The usage of the term."
+            },
+            "validValues": {
+              "type": "array",
+              "description": "An array of related term headers as valid values.",
+              "items": {
+                "$ref": "#/definitions/AtlasRelatedTermHeader"
+              }
+            },
+            "validValuesFor": {
+              "type": "array",
+              "description": "An array of related term headers as valid values for other records.",
+              "items": {
+                "$ref": "#/definitions/AtlasRelatedTermHeader"
+              }
+            }
+          }
+        }
+      ],
+      "description": "The glossary term."
+    },
+    "ResourceLink": {
+      "title": "ResourceLink",
+      "type": "object",
+      "properties": {
+        "displayName": {
+          "type": "string",
+          "description": "Display name for url."
+        },
+        "url": {
+          "type": "string",
+          "description": "web url. http or https"
+        }
+      }
+    },
+    "ContactBasic": {
+      "title": "ContactBasic",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Azure Active Directory object Id."
+        },
+        "info": {
+          "type": "string",
+          "description": "additional information to describe this contact."
+        }
+      }
+    },
+    "TermStatus": {
+      "type": "string",
+      "title": "Status",
+      "description": "Status of the AtlasGlossaryTerm",
+      "enum": [
```

Comment:
All your enum definitions are missing the [x-ms-enum](https://github.com/Azure/autorest/tree/master/docs/extensions#x-ms-enum) attribute.